src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/package-info/test1/package-info.java

```diff
@@ -0,0 +1,5 @@
+/*OK: for test allowing to place annotation before PACKAGE_DEF.*/
+@Deprecated
+package com.puppycrawl.tools.checkstyle.checks.whitespace.test1.packageinfo;
+
+import java.lang.Deprecated;
```

Comment:
yes, ... adding new dependency only  for Inputs is too much.
I am ok to keep usage of Deprecated only. Looks like it is only annotation that could be applied to packages in jdk.